documentation/sphinx/source/api-common.rst.inc

```diff
@@ -326,6 +326,10 @@
 
     If this option has been set more times with this database than the disable option, snapshot reads will *not* see the effects of prior writes in the same transaction. Disabling this option is equivalent to calling |snapshot-ryw-disable-transaction-option| on each transaction created by this database.
 
+.. |option-db-tr-transaction-logging-max-field-length-blurb| replace::
```

Comment:
I think `option-db-tr-logging-max-field-length-blurb` to match the other names

Reply:
Okay, if we're using `transaction-logging` as the name of this kind of thing, I suppose this is fine.